src/test/regress/expected/multi_router_planner_fast_path.out

```diff
@@ -1225,6 +1225,7 @@ DEBUG:  query has a single distribution column value: 1
 (0 rows)
 
 -- create a dummy function to be used in filtering
+SET citus.enable_metadata_sync TO OFF;
```

Comment:
is there a reason for this change?

Reply:
We've a test which depends on non-existency of that function on worker nodes.